src/radical/pilot/agent/scheduler/base.py

```diff
@@ -101,6 +100,8 @@ def finalize_child(self):
         self.publish('command', {'cmd' : 'final',
                                  'arg' : self.cname})
 
+        self._dump_prof()
+
```

Comment:
I may be missing something - but it looks like this will only work for the cont. scheduler?  This probably should have an abstract method in this base class?